src/plugins/data/server/search/strategies/common/async_utils.ts

```diff
@@ -28,6 +28,7 @@ export function getCommonDefaultAsyncSubmitParams(
   }
 ): Pick<
   AsyncSearchSubmitRequest,
+  // @ts-expect-error 'keep_alive' has been removed from the types but still available in the docs
```

Comment:
@JoshMock can you take a look, there are quite a lot of type errors / type removals in this update?